pandas-stubs/core/reshape/melt.pyi

```diff
@@ -1,16 +1,27 @@
+from typing import Hashable
+
 import numpy as np
 from pandas.core.frame import DataFrame
 
+from pandas._typing import HashableT
+
 def melt(
     frame: DataFrame,
     id_vars: tuple | list | np.ndarray | None = ...,
     value_vars: tuple | list | np.ndarray | None = ...,
     var_name: str | None = ...,
-    value_name: str = ...,
+    value_name: Hashable = ...,
     col_level: int | str | None = ...,
     ignore_index: bool = ...,
 ) -> DataFrame: ...
-def lreshape(data: DataFrame, groups, dropna: bool = ..., label=...) -> DataFrame: ...
+def lreshape(
```

Comment:
lreshape is not documented, so remove

Reply:
It is in the top level of pandas.  It is possibly accidentally undocumented. 

```
In [1]: import pandas as pd

In [2]: pd.lreshape
Out[2]: <function pandas.core.reshape.melt.lreshape(data: 'DataFrame', groups, dropna: 'bool' = True, label=None) -> 'DataFrame'>
```

Reply:
`lreshape` is now in the dev docs.  

https://pandas.pydata.org/docs/dev/reference/general_functions.html

Reply:
OK